tar/read.c

```diff
@@ -67,6 +67,8 @@ __FBSDID("$FreeBSD: src/usr.bin/tar/read.c,v 1.40 2008/08/21 06:41:14 kientzle E
 
 #include "bsdtar.h"
 #include "err.h"
+#include "archive_platform.h"
```

Comment:
archive_platform.h is internal to libarchive and must not be used by tar or any other program that uses libarchive.

Why did you find this necessary?